lib/sampatrianon.rb

```diff
@@ -59,7 +60,6 @@ def parse
       parsed_email = @email.body.colons_to_hash(/(Telefone|Origem|Nome|Site|E-mail|Mensagem|Link da Land).*?:/, false)
 
       all_sources = F1SalesCustom::Email::Source.all
-      destinatary = @email.to.map { |email| email[:email].split('@').first }
```

Comment:
Can I remove this line? The variable isn't being used anywhere in this gem's code.